pkgs/development/python-modules/ormar/default.nix

```
@@ -0,0 +1,82 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, databases
+, pydantic
+, sqlalchemy
+, asyncpg
+, psycopg2
+, aiomysql
+, aiosqlite
+, cryptography
+, orjson
+, typing-extensions
+, importlib-metadata
+, aiopg
+, mysqlclient
+, pymysql
+, pytestCheckHook
+, pytest-cov
+, codecov
+, pytest-asyncio
+, fastapi
+}:
+
+buildPythonPackage rec {
+  pname = "ormar";
+  version = "0.10.23";
+  format = "pyproject";
+
```

[Python 3.6](https://github.com/collerek/ormar/blob/master/pyproject.toml#L44) is support but I suggest to set the minimal release to 3.7. This saves a condition for `orjson` later and Python 3.6 is EOL technically.

```suggestion
  disabled = pythonOlder "3.7";
```